### PR TITLE
feat(inline-edit): user should see tooltip upon clicking item cell

### PR DESCRIPTION
**Pull request by ChenCodes**
Changes in this PR:
- Render a Tooltip that wraps a button for a cell in the MultiGrid 

ezgif com-video-to-gif (3)

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @ChenCodes has signed the CLA. Thanks for the pull request!

File: src/features/list-view/ListView.js [hunk 1]
```diff
@@ -2,10 +2,13 @@
 import * as React from 'react';
 import classNames from 'classnames';
 import noop from 'lodash/noop';
+import uniqueId from 'lodash/uniqueId';
+// import { FormattedMessage } from 'react-intl';
```
Comment (ChenCodes) [inline src/features/list-view/ListView.js]: Will uncomment once the relevant code from the parent app is ported over to BUI.

File: src/features/list-view/ListView.js [hunk 2]
```diff
@@ -2,10 +2,13 @@
...
 
 import { MultiGrid } from 'react-virtualized/dist/commonjs/MultiGrid/index';
 import { SORT_ORDER_ASCENDING, SORT_ORDER_DESCENDING } from '../query-bar/constants';
 import IconSortChevron from '../../icons/general/IconSortChevron';
+// import Tooltip from '../../elements/common/Tooltip';
```
Comment (ChenCodes) [inline src/features/list-view/ListView.js]: Same as above

File: src/features/list-view/ListView.js [hunk 3]
```diff
@@ -17,20 +20,30 @@ type Props = {
...
 
 class ListView extends React.PureComponent<Props> {
     cellRenderer = ({ columnIndex, key, rowIndex, style }: CellRendererArgs) => {
-        const { getGridCell, getGridHeader, getGridHeaderSort = noop, onSortChange = noop } = this.props;
-
+        const {
+            getGridCell,
+            getGridHeader,
+            getGridHeaderSort = noop,
+            // onCellClick = noop,
```
Comment (ChenCodes) [inline src/features/list-view/ListView.js]: Same as above

File: src/features/list-view/ListView.js [hunk 4]
```diff
@@ -17,20 +20,30 @@ type Props = {
...
+            // shouldShowTooltip = noop,
```
Comment (ChenCodes) [inline src/features/list-view/ListView.js]: Same as above

**Comment by jstoffan** [comment]
Please do not commit commented code. Deleting and reverting changes as needed is by far the preferable option.

**Comment by ChenCodes** [comment]
Closing this as product has requested a different UX.

**Pull request closed without merging**